SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/SpatialMultiWorkerSettings.h

```diff
@@ -48,6 +50,8 @@ class SPATIALGDK_API UAbstractSpatialMultiWorkerSettings : public UDataAsset
 	UPROPERTY(EditAnywhere, Category = "Multi-Worker")
 	TSubclassOf<UAbstractLockingPolicy> LockingPolicy;
 
+	UPROPERTY(EditAnywhere, Category = "Multi-Worker")
+	bool bEnableHandover{ true };
```

Comment:
any reason for setting here vs. in initializer list with other variables?

Reply:
Nope other than I am a rouge, moved 😄